### ISSUE TITLE
Update to monarc 2.8.0 errors

### ISSUE THREAD

**Issue by eromang** [monarc-project/MonarcAppFO#148]
Dear all,

Update to MONARC 2.8.0 raise critical errors.

1. Home page display warning message, please see screenshot
2. Login no more possible with setup email, account is no more recognised

Best Regards
![ericromang 2019-03-04 at 08 32 30](https://user-images.githubusercontent.com/545923/53717479-963af080-3e58-11e9-81a9-4e82e763ce8d.png)

**Comment by cedricbonhomme** [comment]
Hi,

Could you check that the folder /var/lib/monarc/fo/data exist?
And check the permissions of it:
chown -R www-data:www-data /var/lib/monarc/fo/data , and
chmod -R 770 /var/lib/monarc/fo/data  in order to test

**Comment by eromang** [comment]
/var/lib/monarc/fo/data exist and permissions have been implemented.

The error (1) is still present and login is still not possible.

**Comment by cedricbonhomme** [comment]
I just tested again by migration MONARC 2.7.4 and it worked for me... strange. Did you checked if you also have the folder  /var/lib/monarc/fo/data/cache ?  and try directly with` chmod -R 777 /var/lib/monarc/fo/data/cache`. Because I really think that the errors we can see in your screen shot are the problem.

**Comment by eromang** [comment]
Hello,

I restored Monarc to previous version and started the upgrade process to 2.8 again by following:

$ ./scripts/update-all.sh
$ rm -Rf ./data/cache
$ sudo systemctl restart apache2.service

once update cache directory is no more available creating error number 1
I had to mkdir cache, hown www-data:www-data cache/ and chmod -R 777 /var/lib/monarc/fo/data/cache
Once cache was back again the error disappeared.

Also login was possible again.

Thanks for the support.
